Doc/library/string.rst

```diff
@@ -312,7 +312,7 @@ non-empty format specification typically modifies the result.
 The general form of a *standard format specifier* is:
 
 .. productionlist:: format-spec
-   format_spec: [[`fill`]`align`][`sign`]["z"]["#"]["0"][`width`][`grouping_option`]["." `precision`][`type`]
+   format_spec: [[`fill`]`align`][`sign`]["z"]["#"]["0"][`width`][`grouping_option`]["." `precision` [`grouping_option`]][`type`]
```

Comment:
This is not accurate. More accurate would be ``` ["." `precision` [`grouping_option`] | "." `grouping_option`] ``` or ``` ["." (`precision` [`grouping_option`] | `grouping_option`)] ``` or ``` ["." [`precision`][`grouping_option`]] ``` with addition that either precision or grouping_option should be specified after ".".